src/PrestaShopBundle/Resources/config/services/core/domain/contact.yml

```diff
@@ -0,0 +1,5 @@
+services:
+  _defaults:
+    public: true
```

Comment:
```suggestion
    public: false
```

Reply:
If  you need ot set a service as public, please put it in the service definition, services are meant to be private by default.